strawberry/django/context.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass
+
+from django.http import HttpRequest
+
+
+@dataclass
+class StrawberryDjangoContext:
+    request: HttpRequest
+
+    def __getitem__(self, key):
```

Comment:
just adding a note for future people, we are adding this so that we don't break old code that's using `info.context["request"]`